implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -71,6 +71,18 @@ pub struct CreateCommand {
     )]
     pub tcp_listener_address: String,
 
+    /// The address to bind the UDP listener to. UDP listener is not started unless --udp is passed.
+    /// Once the node is created, its services can be accessed via this address.
+    /// By default, it binds to 127.0.0.1:0 to assign a random free port.
+    #[arg(
+        display_order = 900,
+        long,
+        short,
+        id = "SOCKET_ADDRESS_UDP",
+        default_value = "127.0.0.1:0"
+    )]
+    pub udp_listener_address: String,
```

Comment:
Once we merge https://github.com/build-trust/ockam/pull/8618 we can merge the udp and tcp arguments.